src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,24 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+We need artificats from trustee when do attestation test, to prepare trustee, do as following. 
+
+```
+pushd ${cloud-api-adaptor}/test/e2e
+git clone https://github.com/confidential-containers/trustee.git
+pushd trustee
+git checkout $(../../../hack/yq-shim.sh '.git.kbs.reference' ../../../versions.yaml)
+pushd kbs
+make CLI_FEATURES=sample_only cli
+popd
+popd
+popd
+```
+
+To deploy the KBS service and test attestation related cases, set `export DEPLOY_KBS=yes`.
```

Comment:
Do we need to also add instructions to set:
```
KBS_IMAGE = "ghcr.io/confidential-containers/staged-images/kbs"
KBS_IMAGE_TAG = "dc01f454264fb4350e5f69eba05683a9a1882c41"
```
(which we'd read from the versions.yaml) into the libvirt.properties
or better yet could we get the libvirt e2e to read from the version.yaml itself? Maybe that's a step too far for this PR!

Reply:
@huoqifeng - I think this is the only outstanding comment I have

Reply:
Fixed, thank you @stevenhorsman for the hard work to replay it!